src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/View/ActivityViewBuilderFactory.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ActivityBundle\Infrastructure\Sulu\Admin\View;
+
+use Sulu\Bundle\ActivityBundle\Domain\Model\ActivityInterface;
+use Sulu\Bundle\ActivityBundle\Infrastructure\Sulu\Admin\ActivityAdmin;
+use Sulu\Bundle\AdminBundle\Admin\View\ListViewBuilderInterface;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewBuilderFactoryInterface;
+use Sulu\Component\Security\Authorization\PermissionTypes;
+use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
+
+class ActivityViewBuilderFactory implements ActivityViewBuilderFactoryInterface
+{
+    /**
+     * @var ViewBuilderFactoryInterface
+     */
+    private $viewBuilderFactory;
+
+    /**
+     * @var SecurityCheckerInterface
+     */
+    private $securityChecker;
+
+    public function __construct(
+        ViewBuilderFactoryInterface $viewBuilderFactory,
+        SecurityCheckerInterface $securityChecker
+    ) {
+        $this->viewBuilderFactory = $viewBuilderFactory;
+        $this->securityChecker = $securityChecker;
+    }
+
+    public function createActivityListViewBuilder(
+        string $parent,
+        string $resourceKey,
+        string $resourceIdRouterAttribute = 'id'
+    ): ListViewBuilderInterface {
+        /** @var ListViewBuilderInterface $activityListViewBuilder */
+        $activityListViewBuilder = $this->viewBuilderFactory
+            ->createListViewBuilder($parent . '.activity', '/activities')
```

Comment:
think i would not hardcode the `name` and `path` like this but pass it explicitly like in the `ViewBuilderFactoryInterface`. there might be a usecase where it makes sense to use another path or where there is no parent